### PR TITLE
[skip netlify] kodabot

### LINKED ISSUE #1477: Payout leaderboard for KodaDot
We are all monkeys. 
And would be nice to build trust for external contributors, 
where external contributors could easier justify 
that we do payouts on regular basis and see ppl how they contribute etc
The requirement to run as serverless function, best if we can deploy it in some public way, using autocode.xyz or something like that, should be few lines and cracking graphql and right queries
We'd like to put more emphasis on past records.

### Spec
- make file payout_leaderboard.md of this KodaDot repository
- scrape all PRs where they have kusama.subscan.io URL referring to a particular transaction
- identify developers contributors 
- parse tx and put their balance received on the address, it should be matching what has developer put in the payout link, the format has changed, so mind checking since we've started using a new PR template
- in file show github handle / total amount received / (=total amount recieved/merged PRs) / number of PRs made / merged PRs / closed PRs  / lines added/removed / number of total commits merged / link to last received contribution on subscan
- payout_leaderboard.md should be updated on every PR which was merged with payout to reflect newest numbers.

| show github handle | total amount received | (=total amount recieved/merged PRs) | number of PRs made | merged PRs | closed PRs  | lines added/removed | number of total commits merged | link to last received contribution on subscan |
|-|-|-|-|-|-|-|-|-|

### PR DESCRIPTION
**Pull request by petersopko**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1477, #2448, #2642 
- [x] following #2870, reopened for clarity and update

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by petersopko** [comment]
@yangwao sorry, I saw you've requested review on the old PR I just opened up a new one with updated file to avoid reverting/merge conflicts etc.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,25 +25,117 @@ Before you being:
...
+- In case you want to drop out of the assignment, you can manually unassign yourself
+- If you'd like to get out of the `queue` or `pass` during your `option period` to pick an issue, commenting `pass` or its variation will do the trick (check `goPhrases` below)
+- Once your assignment runs out, KodaBot will unassign you and leave a chance open for other participants
+- Opening PR in time will ensure that you won't get unassigned, and that issue stays yours until the PR gets resolved
+- Getting unassigned, dropping out of the queue, or passing on the option to pick the issue forbids you from further participation in this particular issue.
+- **ignoring issue**: sometimes, you might work on an issue that doesn't need supervision / assigning from a bot or help from the team. As an example, consider a small issue, quick fix, something without a bounty label, etc. In this case, you can comment `ignore` or its variation (check `ignorePhrases` below) to the issue to prevent KodaBot from reacting to this issue. This feature is available only to collaborators on nft-gallery.
+
```
Comment (vikiival) [inline CONTRIBUTING.md:45]: Why not markdown table?

Comment (petersopko) [reply CONTRIBUTING.md:45]: it is markdown table now :eyes:

**Review by vikiival — APPROVED** [review]
LGTM

**Comment by yangwao** [comment]
pay 2000 usd

**Comment by yangwao** [comment]
pay 2000 usd

**Comment by yangwao** [comment]
pay 2000 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $2000 @ 133.68 USD/KSM ~ 14.961 $KSM
🧗 [FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y](https://kusama.subscan.io/account/FNyt7T1xdbhN7dagf7yGYCRJuE3R45VmTCE3tjzy1rKxa7y)
🔗 [0x628cb66f1259c45564a64853a23d0ae83bc93a1cf5459bc16c39e670b83b500c](https://kusama.subscan.io/extrinsic/0x628cb66f1259c45564a64853a23d0ae83bc93a1cf5459bc16c39e670b83b500c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**